sonic_platform_base/sonic_xcvr/api/public/cmis.py

```diff
@@ -267,50 +299,47 @@ def get_transceiver_info(self):
         ext_id = admin_info[consts.EXT_ID_FIELD]
         power_class = ext_id[consts.POWER_CLASS_FIELD]
         max_power = ext_id[consts.MAX_POWER_FIELD]
-
-        xcvr_info = {
+        self.xcvr_info =  copy.deepcopy(CmisApi.cmis_xcvr_info_dict)
```

Comment:
@vvolam I think we should have a function like _get_xcvr_info_default_dict and then implement it to return the corresponding default dictionary.
The current invocation will clear the default dictionary initialized for CCMIS modules.